pkgs/applications/version-management/pijul/pijul.org.context

```diff
@@ -0,0 +1,92 @@
+
+Context:
```

Comment:
Just out of curiosity, what is this file used for? (and what is it generated by?)

Thanks.